src/styles/baseThemes/lightBaseTheme.js

```diff
@@ -27,6 +27,7 @@ export default {
     accent2Color: grey100,
     accent3Color: grey500,
     textColor: darkBlack,
+    secondaryTextColor: fade(darkBlack, 0.7),
```

Comment:
Shouldn't we use `54%`?
![capture d ecran 2016-07-06 a 19 36 55](https://cloud.githubusercontent.com/assets/3165635/16628215/0c3eda62-43b1-11e6-8eb1-18e913dab107.png)